Sources/AppcuesKit/Presentation/Traits/Appcues/AppcuesModalTrait.swift

```diff
@@ -98,3 +118,50 @@ extension AppcuesModalTrait {
         }
     }
 }
+
+@available(iOS 13.0, *)
+extension AppcuesModalTrait.Config {
+    func toTransition() -> AppcuesModalTrait.Transition {
```

Comment:
this is where it defaults to the nearest expected edge, if nothing explicitly configured